client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -23,6 +23,7 @@ import {
 	shouldFetchSitePlans,
 	isDIFMProduct,
 	isPro,
+	isCredits,
```

Comment:
Would you mind ordering those imports in alphabetical order? They are almost ordered already.

```
import {
	isBlogger,
	isBusiness,
	isChargeback,
 	isCredits,
	isDelayedDomainTransfer,
 	isDIFMProduct,
	isDomainMapping,
	isDomainProduct,
	isDomainRedemption,
	isDomainRegistration,
	isDomainTransfer,
	isEcommerce,
	isGSuiteOrExtraLicenseOrGoogleWorkspace,
	isGSuiteOrGoogleWorkspace,
	isJetpackBusinessPlan,
	isJetpackPlan,
	isPersonal,
	isPlan,
	isPremium,
 	isPro,
	isSiteRedirect,
	isStarter,
	isTheme,
	isTitanMail,
 	shouldFetchSitePlans,
} from '@automattic/calypso-products';
```

Reply:
Done! ✅